src/components/landing/faqs.js

```diff
@@ -8,29 +8,35 @@ const FAQ_CONTENT = [{
     "Roadie is SaaS only for the moment. Some Backstage plugins will need a connection back to your infrastructure. We offer a number of secure methods for enabling this.",
   
 }, {
-  id: 1,
+  id: 2,
   question: "Does Roadie support custom plugins?",
   answer:
     "Yes. Roadie provides a private plugin repository which you can publish Backstage plugins too. They can then be manipulated and used just like ordinary open-source Backstage plugins.",
 
 }, {
-  id: 1,
+  id: 3,
   question: "Where is Roadie data stored?",
   answer:
     "All data is stored within the EU and is encrypted at rest using AES-256 encryption algorithm. AWS KMS is used to manage and protect the encryption keys. Data is backed up to AWS S3 at least once per day.",
 
 }, {
-  id: 1,
+  id: 4,
   question: "What level of support does Roadie provide?",
   answer:
     "Every Roadie customer gets a shared Slack or Discord channel. We pride ourselves on our responsiveness. We also deeply value customer feedback and prioritize your requests in our roadmap.",
+},
+{
+  id: 5,
+  question: "Do you have a status page?",
+  answer:
+    "Yes we do! It can be found here: <a href='https://status.roadie.io'>https://status.roadie.io/</a>",
 }];
 
 
 const Item = ({ question, answer }) => (
   <div>
     <dt className="font-semibold text-gray-900">{question}</dt>
-    <dd className="mt-3 text-gray-500">{answer}</dd>
+    <dd className="mt-3 text-gray-500" dangerouslySetInnerHTML={{ __html: answer }}/>
```

Comment:
Sounds scary but i think in this case, its not 'dangerous' as we are only displaying simple text

Reply:
if you make the answer a function, React should be able to render is as JSX.

Reply:
Ok cool thanks!

Reply:
Done

Reply:
you can do something like 
```
{answer instanceOf Function ? asnwer() : answer}
```
so there's no need to necessarily wrap non-html to be html.